src/main/java/com/serheev/serializable/SerializationUtils.java

```
@@ -0,0 +1,105 @@
+package com.serheev.serializable;
+
+import java.io.FileOutputStream;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Set;
+
+public final class SerializationUtils {
+
+    private SerializationUtils() {
+    }
+
+    public static void serializeOrder(Order[] o) {
+        try (ObjectOutputStream oos = new ObjectOutputStream(new FileOutputStream("orders.bin"))) {
+            oos.writeObject(o);
+        } catch (IOException e) {
+            System.err.println("Serialization error: " + e);
+        }
+    }
+
+    public static Order[] deserializeOrder() {
+        try (ObjectInputStream ois = new ObjectInputStream(new FileInputStream("orders.bin"))) {
+            return (Order[]) ois.readObject();
+        } catch (IOException | ClassNotFoundException e) {
+            System.err.println("Deserialization error: " + e);
```

throw new RuntimeException() instead of return null